### PR TITLE
Support multiple recipients

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: lib/ruby_gpg.rb [hunk 1]
```diff
@@ -23,17 +23,23 @@ def encrypt(file, recipient, opts = {})
     ascii = options[:armor] == true ? "-a " : ""
     
     command = "#{gpg_command} #{ascii}--output #{output}" +
-              " --recipient \"#{recipient}\" --encrypt #{file}"
+              " --encrypt #{file}"
+
+    [recipients].flatten.each do |r|
+    command += " --recipient \"#{r}\""
```
Comment (blaix) [inline lib/ruby_gpg.rb:28]: indentation problem here

**Comment by blaix** [comment]
I suspect this breaks the tests. Does it?

Either way, I think this warrants some additions/changes to the tests.

**Comment by raphink** [comment]
I have a branch that rebases it on top of #3 https://github.com/raphink/ruby_gpg/tree/dev/travis_multiple_recipients with builds at https://travis-ci.org/raphink/ruby_gpg/branches

**Comment by blaix** [comment]
Thanks. I've merged the travis branch. I looked at your branch and have a couple comments about it. Can you rebase master and send a pull request?

**Comment by raphink** [comment]
Ok rebase when I get access to a computer again.

**Comment by raphink** [comment]
@blaix #4 fixes this. Closing this one.

**Pull request closed without merging**